system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_endpoint_test.clj

```diff
@@ -75,6 +75,18 @@
                             :Term "SURFACE RADIATIVE PROPERTIES"
                             :VariableLevel1 "REFLECTANCE"}}
             400
+            ["object has missing required properties ([\"concept-ids\"])"]
+
+            "All concept-ids"
+            {:concept-ids ["all" ]
+             :name "TEST NAME 1"
+             :update-field "SCIENCE_KEYWORDS"
+             :update-type "ADD_TO_EXISTING"
+             :update-value {:Category "EARTH SCIENCE"
+                            :Topic "LAND SURFACE"
+                            :Term "SURFACE RADIATIVE PROPERTIES"
+                            :VariableLevel1 "REFLECTANCE"}}
+            400
             ["Concept-ids not found - None provided in the request, and none are associated with the provider-id either."]
```

Comment:
Is this test saying the provider has no collections so specifying "ALL" returns an error?

Reply:
Yes.